policycoreutils/setsebool/setsebool.c

```diff
@@ -130,6 +127,7 @@ static int semanage_set_boolean_list(size_t boolcnt,
 	semanage_bool_key_t *bool_key = NULL;
 	int managed;
 	int result;
+	int enabled = is_selinux_enabled();
```

Comment:
Technically, for consistency, this would be int enabled = (is_selinux_enabled() > 0);
However, it is no longer possible for is_selinux_enabled() to return < 0; that was an artifact of when is_selinux_enabled() used to probe for selinuxfs, so alternatively we could just use is_selinux_enabled() alone in both cases.